src/desktop/components/split_test/README.md

```diff
@@ -1,5 +1,18 @@
 # Split Test
 
+The Split Test component enables the bucketing of users into categories. When a test is enabled, a cookie is set for each user that is also stored on the window via sharify. The cookie/SD var's name is based on the test name, and its value is set to one of the test's possible outcomes. Engineers can expose multiple variations of features, content, components etc. by hooking into a test outcome.
+
+Rolling out a test requires a few steps:
+
+- Add a test with name and outcomes to `running_tests.coffee`
+- Create any feature variations that should be exposed based on outcome
+- Configure analytics events that will be tracked as part of the test
+- Enable the test via a `splitTest.view()` event
+
+The last step is especially important! Users will still receive cookies, and analytics will operate as expected with or without a `splitTest.view()` event, but it is this call that enables the data team to parse analytics in the context of a test.
```

Comment:
Great job calling this out!

Reply:
I add some code snippets in my [PR](https://github.com/artsy/force/pull/3776). We are going to work to consolidate.